[241] Log opened at 2022-12-30 10:33:14
[241] I: Checking remote connect back address.
[241] I: Checking header 'HTTP_X_FORWARDED_FOR'.
[241] I: Checking header 'REMOTE_ADDR'.
[241] W: Remote address not found, connecting to configured address/port: localhost:9000. :-|
[241] W: Creating socket for 'localhost:9000', poll success, but error: Operation now in progress (29).
[241] W: Creating socket for 'localhost:9000', connect: Cannot assign requested address.
[241] E: Could not connect to client. :-(
[241] Log closed at 2022-12-30 10:33:14

